BowerToNPM.java

```diff
@@ -0,0 +1,159 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.StringBuilder;
+import java.nio.file.Path;
```

Comment:
This is a neat hack for Linux and mac systems, but you probably need to use a URL class for Windows users.